### ISSUE TITLE
Historical Reference & Prices Data - Delisted Securities (for Backtesting)

### ISSUE THREAD

**Issue by red-dog32** [iexg/IEX-API#1712]
I want to backtest different value investing strategies, and I'm trying to figure out how to get a list of historical ticker symbols to create a tradable universe from (including delisted tickers). 

For example, if I wanted to test the performance of a strategy that buys the lowest P/E stocks on Jan 1st and rebalances annually from 2010-2020, I would need to know the universe of ticker symbols available to purchase on Jan 1st for each year. Then, I would have to remove all the tickers that are not relevant like ETFs, warrants, preferred shares, SPACs, etc.

IEX Cloud's /ref-data/symbols endpoint does not seem to have tickers that have been delisted. For example, when I filter the data returned from the /symbols endpoint for ticker == SKIS, a company that was acquired in 2019/delisted, it does not return anything. 

However, data for the SKIS symbol is available in other endpoints such as the /company. 

The only affordable data provider I have been able to find that allows you to travel back in time and get retroactive ticker data is Alpha Vantage under their 'Listing and Delisting Status' endpoint. It lets you enter a date as a URL parameter and pulls the active tickers listed on various exchanges at that point in time. However, the data is a mess and is not reliable.

So in summary what I'm looking for is:
1) Historical pricing on delisted securities
2) A reference table of ALL available tickers in the database, not just ones that are active on IEX

Does IEX have this data? Thanks!

**Comment by timpaine-iex** [comment]
duplicate https://github.com/iexg/IEX-API/issues/206
